lib/puppet_x/bodeco/util.rb

```diff
@@ -80,7 +80,13 @@ def initialize(_url, options)
           @proxy_port = uri.port
         end
 
-        ENV['SSL_CERT_FILE'] = File.expand_path(File.join(__FILE__, '..', 'cacert.pem')) if Facter.value(:osfamily) == 'windows' && !ENV.key?('SSL_CERT_FILE')
+        @cacert_file = if options.key?[:cacert_file]
+                         options[:cacert_file]
+                       elsif ENV.key?('SSL_CERT_FILE')
+                         ENV['SSL_CERT_FILE']
+                       elsif Facter.value(:osfamily) == 'windows'
+                         File.expand_path(File.join(__FILE__, '..', 'cacert.pem'))
```

Comment:
This can be simplified:
```suggestion
                         File.join(__dir__, 'cacert.pem')
```

However, is there ever a `cacert.pem` in the directory of this file?

Reply:
Sure, but i did not change that part. Just moved it to a different location in the logic.